src/mjolnir/graphenhancer.cc

```diff
@@ -94,58 +96,249 @@ struct enhancer_stats {
   }
 };
 
+/*
+The json basically looks like this:
+
+[{
+  "iso3166-1": "us",
+  "iso3166-2": "pa",
+  "urban": {
+    "way": [1,2,3,4,5,6,7,8],
+    "link_exiting": [9,10,11,12,13,14],
+    "link_turning": [15,16,17,18,19,20],
+    "roundabout": [21,22,23,24,25,26,27,28],
+    "driveway": 29,
+    "alley": 30,
+    "parking_aisle": 31,
+    "drive-through": 32
+  },
+  "rural": {
+    "way": [33,34,35,36,37,38,39,40],
+    "link_exiting": [41,42,43,44,45,46],
```

Comment:
@kevinkreiser we don't have any TC or links less than kTertiary

Reply:
when i scanned my latest planet i did find a very small number of `residential_link`s but i think you're right the number is so low its not worth it. thanks for pointing that out :+1:

Reply:
@kevinkreiser hmmm confused a bit.  based on this https://github.com/valhalla/valhalla/blob/master/src/mjolnir/linkclassification.cc#L231 the lowest classification for *_link would be `kTertiary`

Reply:
there were 10 links in my planet file that had a class of residential. tbh im not surprised that reclassification didnt touch them. they are already a low class road which means the reclassifier wouldnt want to touch them anyway since they are already low classed.

but in any case ive removed this from the configuration as you suggested. also its important to note that im not trying to build this config to match valhallas schema, im trying to build it to match OSM in general so that any routing project can use it. the final home for this format will be determined over here: https://github.com/OpenStreetMapSpeeds